style.css

```diff
@@ -1,42 +1,79 @@
-/*
- * Mums in Tech | Week 03
- *
- * I have put some basic CSS in place to give everyone a
- * nice starting point. Feel free to change it however.
- *
- * You challenge, if you choose to accept it, is to
- * put your own design/ideas into the provided HTML with
- * minimal changes to the HTML itself.
- */
-
 body {
-  /* I prefer sans-serif (without curls) so I'm changing it haha */
-  font-family: Helvetica, sans-serif;
+  font-family: "Comic Sans MS", cursive, sans-serif;
+  background-color: #a9c3a8;
+  background-image: url("http://sg.theasianparent.com/wp-content/blogs.dir/1/files/7-mothers-ruling-the-tech-world/we-can-do-it.jpg");
+  background-repeat: no-repeat;
+  background-position: left center;
+  background-attachment: fixed;
+  line-height: 1.3;
```

Comment:
I believe its best to list the unit here so 1.3 likely means 1.3 `em` which is a unit relative to the base font size of the page (if none is set then its relative to the browsers default font size). I find it always a good setting to use though as it makes reading content much easier to have a nice spacing between lines.